node/actors/consensus/Cargo.toml

```diff
@@ -1,27 +1,26 @@
 [package]
-name = "consensus"
+name = "zksync_consensus_consensus"
```

Comment:
Agree that the naming looks weird, but I think having inconsistent naming is worse.

Reply:
Why would it be inconsistent, provided that the crate directory is renamed as well? Not insisting on renaming, just curious. One could argue that `bft` is a more specific suffix than `consensus` because it specifies the kind of consensus in terms of security assumptions (i.e., validators can behave arbitrarily).

Reply:
If the crate directory is also renamed then I'm in favor. I just didn't want the crate and the directory to have different names.